library/Zend/InputFilter/InputInterface.php

```diff
@@ -26,6 +26,13 @@ public function setAllowEmpty($allowEmpty);
      */
     public function setBreakOnFailure($breakOnFailure);
 
+
+    /**
+     * @param bool $continueIfEmpty
+     * @return self
+     */
+    public function setContinueIfEmpty($continueIfEmpty);
+
```

Comment:
This is defined in `EmptyContextInterface`, and is there because we cannot add to existing interfaces (causes a BC break). This is why `Input` implements `EmptyContextInterface`.